CHANGELOG.md

```diff
@@ -23,6 +23,9 @@ Main (unreleased)
 
 - Add metrics when clustering mode is enabled. (@rfratto)
 
+- Fix spelling of the `frequency` argument on the `local.file` component.
+  (@tpaschalis)
+
```

Comment:
Technically this is a breaking change?

Reply:
Well, maybe yes? It could also be a bug since it _now_ matches what the configuration used to call the field.

We may have to document this on our Flow-specific [upgrade guide page] (https://grafana.com/docs/agent/next/flow/upgrade-guide/). It might be a good conversation with the team on how we handle such cases.

Reply:
> It might be a good conversation with the team
Agreed - also how does it work with components being in beta. 

In practice, if I have a config that passes the old spelling, I will get an error on upgrade, so I think it's worth highlighting this in docs in one way or the other.

Reply:
(not disagreeing here, just playing devil's advocate) it wouldn't even be possible to get a config that passes just by reading the docs, you'd have to delve into the code.

Reply:
Right, the docs were correct all the time. Makes sense to keep this as a bugfix then :) LGTM